packages/istanbul-reports/lib/html/annotator.js

```diff
@@ -164,6 +164,18 @@ function annotateBranches(fileCoverage, structuredText) {
                     gt;
                 closeSpan = lt + '/span' + gt;
 
+                if (
```

Comment:
I think this deserves a through comment, otherwise this looks good to me 😄

Reply:
Ok, I added a comment.